modules/ai/assets/js/editor/pages/form-media/components/generate-submit.js

```diff
@@ -1,11 +1,14 @@
 import { Button } from '@elementor/ui';
 
 const GenerateImagesSubmit = ( props ) => {
+	const label = props.children ?? __( 'Generate', 'elementor' );
```

Comment:
You can remove this line + please change the function name to `GenerateSubmit` (my bad).